### PR TITLE
added multiplication function to frontend numpy

### PR DESCRIPTION
**Pull request by pushpahasa1**
multiply 

- [ ] #2492

### CODE REVIEW

**Comment by pushpahasa1** [comment]
please let me know if any corrections needed

File: ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py [hunk 1]
```diff
@@ -48,3 +48,28 @@ def subtract(
 
 
 subtract.unsupported_dtypes = {"torch": ("float16",)}
+
+#multiply
```
Comment (rush2406) [inline ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py:52]: can be removed

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py [hunk 2]
```diff
@@ -110,3 +110,42 @@ def test_numpy_subtract(
         subok=True,
         test_values=False,
     )
+    
+    
+#multiply
```
Comment (rush2406) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py:115]: can you please add the data generation part for hypothesis? you can follow similar to [example](https://github.com/unifyai/ivy/blob/fe1e0d78073ddad93162ece2cee4a8269985e5ca/ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py#L12)

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @pushpahasa1 
Thank for contributing! Please add the hypothesis generation part as explained in the comment. Also please fix the lint issues

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @pushpahasa1 
Thanks for the changes! Can you please resolve the conflicts and let me know, remaining looks good to me!

**Comment by pushpahasa1** [comment]
@rush2406 I have added non conflicting file above under "conflict removed"

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @pushpahasa1 
The conflict still seems to exist, so I can't merge to master. Can you try pasting the master copy of the file and then paste your changes? maybe that can help?

**Comment by pushpahasa1** [comment]
@rush2406  please check now.

**Comment by pushpahasa1** [comment]
Hi @rush2406 one thing,  I couldn't synchronise my forked repository. If i synchronise it will delete all existing comments made from me. Is it that because I haven't updated my repo the merge conflict is occurring, due to updates happened in other file in ivy repo but not updated in my repo?
If so please let me know I will send a fresh PR in that case

**Comment by pushpahasa1** [comment]
Hi @rush2406 Any updates?

**Comment by pushpahasa1** [comment]
Hi @rush2406 Due to rapid updates happening in codebase I noticed  there is new conflict happening in test generation part. I updated it. To avoid future merge conflict I request you to merge before any new updates happening on codebase. 
Thank you

**Comment by rush2406** [comment]
@pushpahasa1 no worries, I have resolved the new conflicts that have come up and merge the PR, thanks!

**Pull request merged**